objects/_objects.layout.scss

```diff
@@ -211,6 +211,10 @@ $inuit-use-markup-fix: false !default;
     display: flex;
   }
 
+  &.o-layout--center {
+    justify-content: center;
+  }
```

Comment:
This notation is a bit awkward, to be honest. It’d only be available when `o-layout--stretch` is set on the same element. This to me is a bit to unflexible. By the way I find it a bit weird to have `o-layout--stretch o-layout--center` on the same element. I’d assume both are for different things, not an add-up class.